modules/n1ql/pages/n1ql-language-reference/index-partitioning.adoc

```diff
@@ -596,7 +595,7 @@ For example:
 * A cluster has a mix of non-partitioned indexes and partitioned indexes.
 * There is data skew in the partitions.
 
-In Couchbase Server 7.0 and later, the [def]_index redistribution_ setting enables you to specify how Couchbase Server redistributes indexes on rebalance.
+In Couchbase Capella, the [def]_index redistribution_ setting enables you to specify how Couchbase Capella redistributes indexes on rebalance.
 For further details, refer to {rebalancing-the-index-service}[Rebalancing the Index Service].
```

Comment:
Need to replace this link with link to the Server side, unless there's anything about rebalance in the Capella docs?

Reply:
[Updated to a Capella link above](https://github.com/couchbaselabs/docs-devex/pull/21/files#diff-0ade0c2e6945b6444633b2494ea996d76c848334a2267330500974c643b6f24fR8).